src/decoders/router_decoder_custom_worker.erl

```diff
@@ -74,17 +74,45 @@ init(Args) ->
     ID = maps:get(id, Args),
     VM = maps:get(vm, Args),
     Function = maps:get(function, Args),
-    Context = init_context(VM, Function),
-    TimerRef = erlang:send_after(?TIMER, self(), timeout),
-    %% self() ! ?INIT_CONTEXT,
-    State = #state{
-        id = ID,
-        vm = VM,
-        context = Context,
-        function = Function,
-        timer = TimerRef,
-        backoff = backoff:type(backoff:init(?BACKOFF_MIN, ?BACKOFF_MAX), normal)
-    },
+    State =
+        case init_context(VM, Function) of
+            {ok, Context} ->
+                TimerRef = erlang:send_after(?TIMER, self(), timeout),
+                #state{
+                    id = ID,
+                    vm = VM,
+                    context = Context,
+                    function = Function,
+                    timer = TimerRef,
+                    backoff = backoff:type(backoff:init(?BACKOFF_MIN, ?BACKOFF_MAX), normal)
```

Comment:
None of the backoff stuff is being used, can we drop it?

Reply:
If we drop all of the backoff functionality, yes it will keep the code simple, and then yes we can remove those chunks of commented-out code.

(The original motivation for backoff was to avoid a tight loop to retry creating a context when that initial call fails.  However, now with better understanding, a failed context should be sufficiently rare that there's little value gained from using the backoff library here.)

Reply:
Fixed in commit 731a85b